src/components/Output/custom-els/RangeInput/index.ts

```diff
@@ -0,0 +1,94 @@
+import './styles.css';
+
+const EVENTS = [
+  'input',
+  'change',
+  'focus',
+  'blur',
+];
+
+const PROPERTIES = [
+  'name',
+  'value',
+  'min',
+  'max',
+  'step',
+];
+
+interface RangeInputElement {
+  value: string;
+  min?: string;
+  max?: string;
+  step?: string;
```

Comment:
None of `min`, `max`, or `step` are optional.

Reply:
They have default values, but I agree it's likely pointless to support that.